src/languages/es.js

```diff
@@ -517,6 +517,9 @@ export default {
         listOfRestrictedBusinesses: 'lista de negocios restringidos',
         incorporationDatePlaceholder: 'Fecha de inicio (aaaa-mm-dd)',
         companyPhonePlaceholder: '10 dígitos, sin guiones',
+        confirmModalTitle: '¿Está seguro?',
```

Comment:
```suggestion
        confirmModalTitle: '¿Estás seguro?',
```

Reply:
These changes are because we use the informal tu instead of the formal usted

Reply:
Changed. Thanks for informing.